SPECS/sed/sed.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Stream editor
 Name:           sed
 Version:        4.9
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
This is a test change to see if check-manifests pipeline (PR check) is working correctly